airgun/session.py

```diff
@@ -231,11 +232,14 @@ def _open(self, entity):
             else:
                 endpoint = getattr(entity, 'endpoint_path', '/')
             full_url = f"https://{settings.satellite.hostname}{endpoint}"
-            self._prepare_browser(full_url)
+            if entity.__name__ == "RHSSOLoginEntity":
+                self._prepare_browser(full_url, login_type="RHSSO")
+            else:
+                self._prepare_browser(full_url)
```

Comment:
For consistency, it's better to add that to object instance as attribute instead of passing around as parameter.

Also, how many login types do we plan to support in near future? That's the difference between naming attribute `login_type` (which suggests at least three different methods, one of which must be used exclusively) and `rhsso_login` (which would be boolean: we either use RHSSO or we don't).